src/components/TopBar/TopBar.vue

```diff
@@ -257,6 +281,30 @@ export default {
 				'--original-color-main-background': window.getComputedStyle(document.body).getPropertyValue('--color-main-background')
 			}
 		},
+
+		nextEvent() {
+			return this.chatExtrasStore.getNextEvent(this.token)
+		},
+
+		eventInfo() {
+			return this.nextEvent ? moment(this.nextEvent.start * 1000).calendar() : null
+		},
+
+		showUpcomingEvent() {
+			return this.nextEvent && !this.isInCall && !this.isSidebar && !this.isMobile
+		},
+	},
+
+	watch: {
+		token: {
+			immediate: true,
+			handler(value) {
+				if (!value || this.isInCall || this.isSidebar) {
```

Comment:
```suggestion
				if (!value || this.isSidebar) {
```

We can't jump straight to the call, so would be always falsy. Even if that would change in future, it's better to request the information to show later